source/dofs/dof_handler.cc

```diff
@@ -2511,7 +2489,9 @@ DoFHandler<dim, spacedim>::distribute_dofs(
          ExcMessage("The Triangulation you are using is empty!"));
   Assert(ff.size() > 0, ExcMessage("The hp::FECollection given is empty!"));
 
+  //
   // register the new finite element collection
+  //
```

Comment:
?

Reply:
I wanted to highlight these comments as they describe the whole following code block and not just a few upcoming lines. I felt like this gives a better overview over the structure of this function.

I don't have a problem with those lines being removed. Do you want them to be removed?